flake.nix

```diff
@@ -7,20 +7,10 @@
 
     holonix = {
       url = "github:holochain/holonix?ref=main-0.4";
-      inputs = {
```

Comment:
This is something that has been presented to me by the people who knew more about Nix than me as a good thing to do. So that you only build with one copy of nixpkgs and other inputs. Sounds great, in reality, it breaks caching all the time.

I noticed that some things that are definitely cached on Cachix by the Holonix CI are being rebuilt here.